tests/end2end/features/utilcmds.feature

```diff
@@ -164,3 +164,7 @@ Feature: Miscellaneous utility commands exposed to the user.
     Scenario: Using debug-log-filter with invalid filter
         When I run :debug-log-filter blah
         Then the error "filters: Invalid value blah - expected one of: statusbar, *" should be shown
+
+    Scenario: Using debug-log-filter
+        When I run :debug-log-filter webview
+        Then no crash should happen
```

Comment:
Right - I'm fine with that for the record FWIW if nothing else works.

What could work is to immediately reset the filter after setting it (and maybe testing if it works, with `:pyeval` or so maybe?), and initially setting it to something which still outputs enough for the test infrastructure to work (something like `ipc,command` or so)